frontend/src/metabase/routes.jsx

```diff
@@ -241,7 +241,10 @@ export const getRoutes = store => (
           <Route path=":slug/:objectId" component={QueryBuilder} />
         </Route>
 
-        <Route path="/model/:slug/detail" component={ModelDetailPage} />
+        <Route path="/model/:slug/detail">
+          <IndexRoute component={ModelDetailPage} />
+          <Route path=":tab" component={ModelDetailPage} />
```

Comment:
It seems that we usually try not to use a generic route pattern when it's actually static. Can we refactor this to static routes with corresponding components, like:
```
<Route path="usage" component={ModelUsagePage} />
<Route path="actions" component={ModelActionsPage} />
...
```
?

This way you won't have to deal with "unknown" tabs and checking whether actions are enabled would be straightforward. WDYT?

Reply:
I tried this yesterday but ran into something that made me switch to this 🤔 Let me try again just to make sure tho

Reply:
Nvm, whatever the problem was it's no longer relevant, switched to explicit tab definitions 👍